### ISSUE TITLE
oc rsh doesn't work for openshift-3-9 image

### ISSUE THREAD

**Issue by GrahamDumpleton** [openshift-labs/learn-katacoda#233]
In 3.9 playground where using openshift-3-9 image, ``oc rsh`` doesn't work.

```
$ oc rollout status dc/blog
replication controller "blog-1" successfully rolled out

$ oc get pods
NAME           READY     STATUS    RESTARTS   AGE
blog-1-nbt4s   1/1       Running   0          52s
$ oc rsh blog-1-nbt4s
error: unable to upgrade connection: <!DOCTYPE html>
<html lang=en>
  <meta charset=utf-8>
  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">
  <title>Error 400 (Bad Request)!!1</title>
  <style>
    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}#logo{background:url(//www.google.com/images/branding/googlelogo/1x/googlelogo_color_150x54dp.png) no-repeat;margin-left:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat;-webkit-background-size:100% 100%}}#logo{display:inline-block;height:54px;width:150px}
  </style>
  <a href=//www.google.com/><span id=logo aria-label=Google></span></a>
  <p><b>400.</b> <ins>That’s an error.</ins>
  <p>Your client has issued a malformed or illegal request.  <ins>That’s all we know.</ins>
```

Means I can't upgrade older 3.6 scenarios which uses ``oc rsh``.

**Comment by GrahamDumpleton** [comment]
Can test problem by using scenario:

* https://katacoda.com/graham.dumpleton/courses/introduction/transferring-files

**Comment by BenHall** [comment]
This was due to websockets protocol not being supported by our proxy. This looks to have been fixed and is working with 3.11

```
$ oc rsh -n openshift-web-console webconsole-7df4f9f689-wzthw
sh-4.2$
```
